src/components/Table/head/header.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable no-param-reassign,class-methods-use-this */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { uniqueId } from '../../../libs/utils';
+import RenderIf from '../../RenderIf';
+import ArrowDown from './arrowDown';
+
+function getClassName(sortable, selected) {
+    return classnames(
+        'rainbow-table_header',
+        {
+            'rainbow-table_header--sortable': sortable,
+            'rainbow-table_header--selected': selected,
+        },
```

Comment:
this is right but if we always put this inside the class why here it is outside?